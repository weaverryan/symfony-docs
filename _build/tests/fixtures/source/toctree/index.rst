
.. toctree::

    file

.. toctree::
    :maxdepth: 1

    file

.. toctree::

    directory/another_file

.. toctree::
    :glob:

    *

.. toctree::
    :glob:

    file
    *

.. toctree::
    :hidden:

    file
