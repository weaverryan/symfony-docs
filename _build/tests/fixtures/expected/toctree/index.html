<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<h1 id="toctree">Toctree<a class="headerlink" href="#toctree" title="Permalink to this headline">Â¶</a></h1>
<div class="toctree-wrapper compound">
    <ul>
        <li><a class="reference internal" href="file.html">Title</a>
            <ul>
                <li><a class="reference internal" href="file.html#sub-title">Sub title</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="toctree-wrapper compound">
    <ul>
        <li><a class="reference internal" href="file.html">Title</a></li>
    </ul>
</div>
<div class="toctree-wrapper compound">
    <ul>
        <li><a class="reference internal" href="directory/another_file.html">Another file</a></li>
    </ul>
</div>
<div class="toctree-wrapper compound">
    <ul>
        <li><a class="reference internal" href="index.html">Toctree</a></li>
        <li><a class="reference internal" href="directory/another_file.html">Another file</a></li>
        <li><a class="reference internal" href="file.html">Title</a>
            <ul>
                <li><a class="reference internal" href="file.html#sub-title">Sub title</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="toctree-wrapper compound">
    <ul>
        <li><a class="reference internal" href="file.html">Title</a>
            <ul>
                <li><a class="reference internal" href="file.html#sub-title">Sub title</a></li>
            </ul>
        </li>
        <li><a class="reference internal" href="index.html">Toctree</a></li>
        <li><a class="reference internal" href="directory/another_file.html">Another file</a></li>
    </ul>
</div>
</body>
</html>
